src/Features/VisualBasic/Portable/Diagnostics/Analyzers/TypeSyntaxSimplifierWalker.vb

```diff
@@ -8,11 +9,40 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeFixes.SimplifyTypeNames
     Friend Class TypeSyntaxSimplifierWalker
         Inherits VisualBasicSyntaxWalker
 
+        Private Shared ReadOnly s_emptyAliasedNames As ImmutableHashSet(Of String) = ImmutableHashSet.Create(Of String)(CaseInsensitiveComparison.Comparer)
+
+        Private Shared ReadOnly s_predefinedTypeMetadataNames As ImmutableHashSet(Of String) = ImmutableHashSet.Create(
+            CaseInsensitiveComparison.Comparer,
+            NameOf([Boolean]),
+            NameOf([SByte]),
+            NameOf([Byte]),
+            NameOf(Int16),
+            NameOf(UInt16),
+            NameOf(Int32),
+            NameOf(UInt32),
+            NameOf(Int64),
+            NameOf(UInt64),
+            NameOf([Single]),
+            NameOf([Double]),
+            NameOf([Decimal]),
+            NameOf([String]),
+            NameOf([Char]),
+            NameOf(DateTime),
+            NameOf([Object]))
```

Comment:
consider writin these all as `NameOf(System.Whatever)`.  Same result, but it will read nicer, without the need for the escaping :)

Reply:
That just leaves it suggesting simpler names. I'll keep it like this for now.